More Vox Sounds

TrixxedHeart opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
Adds a handful of new Vox sounds (Crying, Deathgasp, Cough, and Sigh)

## Why / Balance
i want to be more annoying :godo:

To be serious though, the sounds of a species are one of the key factors that makes them unique in my eyes. Vox lacking most other sounds besides laughing and screaming (not including talking) always felt a bit upsetting imo.

This also helps Vox with the gasp sound, giving a more clear indicator to themselves and other players they need more nitrogen.

## Technical details
Adds 5 new sounds for Vox
- Deathgasp
- Cough
- Cry
- Gasp
- Sigh

All files _should_ be in mono, I double triple checked.

Adds missing attribution file to Vox sounds and fixes incorrect attribution to the shriek1.ogg sound effect and corrects licenses. The shriek1.ogg sound was being incorrectly attributed to a repo that didn't even have the sound. The earliest version I found was from Paradise, not Goon.

## Media

https://github.com/user-attachments/assets/74fd6e33-103a-41cc-807b-b5d016016eeb




https://github.com/user-attachments/assets/aaf14b75-5d82-4460-b19b-5b81e41cc8b4



## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

**Changelog**
:cl:
- add: Added 5 new Vox sounds, (Death gasp, Cough, Cry, Gasp, and Sigh)

FairlySadPanda commented on Resources/Prototypes/Voice/speech_emote_sounds.yml:
```diff
@@ -226,12 +226,22 @@
+    Gasp:
+      path: /Audio/Voice/Vox/vox_gasp.ogg
+    DefaultDeathgasp:
+      path: /Audio/Effects/Gasp/vox_DeathGasp.ogg
```
This file name is weird compared to the others.

arenaconspiracy commented:
One must imagine vox happy before ~~GTA 6~~ more moth/reptilian sounds.

TrixxedHeart commented:
> I believe there's another PR for species sounds that has Vox ones, but these are pretty good. The other PR is merge ready but has been stuck in awaiting merge hell

It has licensing issues and also has been broken for nearly a month
![image](https://github.com/user-attachments/assets/84d68212-75fe-4a64-9b17-74ff799ea437)

MissKay1994 commented:
> It has licensing issues and also has been broken for nearly a month
> _(also personally I think it over does some of the Vox sounds a bit)_

I can guess what happened, but F for that one. I liked some of those, so I love this one more

TrixxedHeart commented:
Quick findings/update to this:
Attribution for shriek1.ogg (which i originally copied from the talk attribution.yml) was incorrect, there is seemingly _no_ shriek1.ogg that has ever existed in the goonstation repo. The earliest version of this sound effect I could find was actually from Paradise Station. I've corrected the attributions for the ones this PR adds as well as the talk attributions.

TrixxedHeart commented on Resources/Audio/Voice/Vox/attributions.yml:
```diff
@@ -0,0 +1,30 @@
+- files:
+  - vox_laugh.ogg
+  license: "CC-BY-SA-3.0"
+  copyright: "Made by Errant."
```
updated

SlamBamActionman reviewed (changes requested):
I am in strong favor of this PR overall since it adds new Vox sounds that I think are in line with the existing Vox soundscape. 

My *one* issue is that the sigh and deathgasp are really close in length and cadence. I could easily see them get mixed up. Just a suggestion, but maybe for the deathgasp extend the start half a second before the wobble effect comes in?

TrixxedHeart commented:
> I am in strong favor of this PR overall since it adds new Vox sounds that I think are in line with the existing Vox soundscape.
> 
> My _one_ issue is that the sigh and deathgasp are really close in length and cadence. I could easily see them get mixed up. Just a suggestion, but maybe for the deathgasp extend the start half a second before the wobble effect comes in?

I'm not great with audio work so I kinda tried to take a slightly different approach

https://github.com/user-attachments/assets/5268d80d-3ea8-48d5-9412-b6843847446d

To be specific, I adjusted it to make the deathgasp have a more distinct and sharper pitch down, while also just being lower pitch in general

I also changed the sigh to start from high pitch start and made it slightly faster to distinguish it more from the deathgasp

SlamBamActionman reviewed (approved):
Sounds good to me! Love it!
